src/Application/Regulation/View/Measure/LocationView.php

```diff
@@ -7,10 +7,13 @@
 final readonly class LocationView
 {
     public function __construct(
-        public string $cityLabel,
+        public ?string $cityLabel,
         public ?string $roadName,
+        public ?string $roadType,
```

Comment:
(Détail) Je chipote mais ailleurs le roadType est en premier

```suggestion
        public ?string $roadType,
        public ?string $cityLabel,
        public ?string $roadName,
```